### PR TITLE
Add missing colon in JSON object

### PR DESCRIPTION
**Pull request by nhducit**

### CODE REVIEW

File: content/examples.md [hunk 1]
```diff
@@ -301,8 +301,8 @@ even if the pattern as a whole matches a node further down the tree.
 ``` js
 // JSON
 {"people": [
-   {"name":"Baz", "age":34, "email": "baz@example.com"}
-   {"name":"Boz", "age":24}
+   {"name":"Baz", "age":34, "email": "baz@example.com"},
+   {"name":"Boz", "age":24},
    {"name":"Bax", "age":98, "email": "bax@example.com"}}
```
Comment (JuanCaicedo) [inline content/examples.md]: Could you remove the second `}` on this line as well?

Comment (nhducit) [inline content/examples.md]: @JuanCaicedo sure

Comment (JuanCaicedo) [inline content/examples.md]: Thanks!

**Comment by JuanCaicedo** [comment]
@jimhigson, this looks good to go, it's updating the example code, so I don't think it needs to be tested

**Pull request merged**